push-webapp/dist/index.js

```diff
@@ -55535,6 +55536,7 @@ async function run() {
     const service = core.getInput('service');
     const indexFile = core.getInput('indexFile');
     const templated = core.getInput('templated') === 'true';
+    const compressed = core.getInput('compressed');
```

Comment:
I think you need to add this to the inputs in action.yml as well: https://github.com/BrandwatchLtd/slipstream-actions/blob/35a7d5e195447bdccba0364751f63e7f23efb55f/push-webapp/action.yml#L3